omod/src/main/java/org/openmrs/module/registrationapp/fragment/controller/MatchingPatientsFragmentController.java

```diff
@@ -178,7 +178,7 @@ private void addToPatient(Patient patient, AppDescriptor app, PersonName name, P
 
     private Boolean alreadyInResults(Patient patient, List<SimpleObject> results) {
         for (SimpleObject result : results) {
-            if (Integer.valueOf(result.get("patientId").toString()).equals(patient.getId())) {
+            if (result.get("uuid").toString().equals(patient.getUuid())) {

```

Comment:
thanks @icrc-psousa ,what steps can i use to reproduce the issue?

Reply:
hi @HerbertYiga, there's steps to reproduce on qa-ref-app on ticket description.